package.json

```diff
@@ -24,6 +24,17 @@
   "eslintConfig": {
     "extends": "react-app"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged"
+    }
+  },
+  "prettier": {
```

Comment:
Thats the config for prettier, a javascript / css / etc code formatting tool. I installed it in that way that it'll automatically format all the code you will commit.